src/content/en/updates/2018/03/webauthn-credential-management.md

```diff
@@ -0,0 +1,175 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: WebAuthn may break your website on Firefox and Edge if you are using the Credential Management API 
+
+{# wf_published_on: 2018-03-09 #}
+{# wf_updated_on: 2018-03-09 #}
+{# wf_featured_image: /web/updates/images/generic/security.png #}
+{# wf_tags: credentials,webauthn,sign-in #}
+{# wf_featured_snippet: WebAuthn may break your website on Firefox and Edge if you are using the Credential Management API #}
+{# wf_blink_components: N/A #}
+
+# WebAuthn may break your website on Firefox and Edge if you are using the Credential Management API {: .page-title }
+
+{% include "web/_shared/contributors/agektmr.html" %}
+
+<div class="clearfix"></div>
+
+## TL;DR
+[WebAuthn](https://www.w3.org/TR/webauthn/) helps increase security by bringing
+public-key credential based authentication to the Web, and is soon to be
+supported in Chrome, Firefox and Edge ([with the updated
+spec](https://docs.microsoft.com/en-us/microsoft-edge/dev-guide/device/web-authentication#differences-between-microsoft-edge-and-the-spec)).
+But this may break websites using [the Credential Management
+API](https://www.w3.org/TR/credential-management-1/) on Firefox and Edge.
```

Comment:
nit: break your website on Firefox and Edge if you are using the Credential Management API to access password-based and/or federated credentials.